src/pages/organizers/[organizerId]/preview/index.page.tsx

```diff
@@ -0,0 +1,267 @@
+                  closable
+                  fullWidth
+                  onClose={() => {
+                    setIsSuccessAlertVisible(false);
+                  }}
+                >
+                  <Trans
+                    i18nKey="organizers.ownerships.request.success"
+                    values={{
+                      organizerName: organizerName,
+                    }}
+                  />
+                </Alert>
+              )}
+              {isErrorAlertVisible && (
+                <Alert
+                  variant={AlertVariants.DANGER}
+                  marginBottom={4}
+                  fullWidth
+                  closable
+                  onClose={() => {
+                    setIsErrorAlertVisible(false);
+                  }}
+                >
+                  <Trans i18nKey="organizers.ownerships.request.confirm_modal.error" />
+                </Alert>
+              )}
+
+              <Inline spacing={5}>
+                <Stack flex={3}>
+                  <OrganizerTable organizer={organizer} />
+                </Stack>
+                <Stack spacing={3.5} flex={1}>
+                  {!canEdit && isOwnershipEnabled && !isOwnershipRequested && (
+                    <Button
+                      variant={ButtonVariants.PRIMARY}
+                      onClick={() => setIsQuestionModalVisible(true)}
+                    >
+                      {t('organizers.detail.actions.request')}
+                    </Button>
+                  )}
+                  {canEdit && (
+                    <Button
+                      variant={ButtonVariants.SECONDARY}
+                      spacing={3}
+                      iconName={Icons.PENCIL}
+                      onClick={() =>
+                        router.push(`/organizer/${organizerId}/edit`)
+                      }
+                    >
+                      {t('organizers.detail.actions.edit')}
+                    </Button>
+                  )}
+                  {canEdit && isOwnershipEnabled && !isOwnershipRequested && (
+                    <Button
+                      variant={ButtonVariants.SECONDARY}
+                      spacing={3}
+                      onClick={() =>
+                        router.push(`/organizer/${organizerId}/ownerships`)
+                      }
+                    >
+                      {t('organizers.detail.actions.manage')}
+                    </Button>
+                  )}
+                  <Button
+                    variant={ButtonVariants.SECONDARY}
+                    spacing={3}
+                    iconName={Icons.ARROW_LEFT}
+                    onClick={() => router.push(`/organizers`)}
+                  >
+                    {t('organizers.detail.actions.back')}
+                  </Button>
+                </Stack>
+              </Inline>
+            </Stack>
+          )}
+          {!isOwnershipEnabled && <OrganizerTable organizer={organizer} />}
```

Comment:
Would maybe restructure the HTML a bit so that the common parts are rendered either way, here since the table will always be here I would move it out of either condition and only conditionally render the ownerships-related parts, to also reduce the indentation levels here since it can be confusing that most of the rendering is done in a condition